roles/tikv/templates/tikv.toml.j2

```diff
@@ -30,7 +30,15 @@
 
 [storage]
 {% for item, value in tikv_conf.storage | dictsort -%}
+{% if value is not mapping -%}
 {{ item }} = {{ value | to_json }}
+{% else %}
+
+[storage.{{ item }}]
+{% for sub_item, sub_value in value | dictsort -%}
+{{ item }} = {{ value | to_json }}
```

Comment:
I'm not familiar with it, but I think it should be `{{ sub_item }} = {{ sub_value | to_json }}` ?

Reply:
thanks!